Resources/Prototypes/Body/Organs/moth.yml

```diff
@@ -4,12 +4,16 @@
   categories: [ HideSpawnMenu ]
   components:
   - type: Stomach
-    # specialDigestible: # Frontier
-    #   tags: # Frontier
-    #   - MothFood # Frontier
-    #   - Fruit # Frontier
-    #   - Pill # Frontier
-    #   - Crayon # Frontier
+    specialDigestible:
```

Comment:
By allowing moths to eat only MothFood, this _severely_ restricts moth players. With no ability to eat fruit or pills, that leaves moths with like 4 foods. If there's no food ship around (think lowpop), this is considerably less than ideal. On upstream, moths can at least eat easily-obtained jumpsuits (which  I'm not advocating for here). Given the supremely limited selection of moth-friendly nommables, I think we should reconsider this. I don't think this will lead to better or more interesting roleplay; I think it'll lead to a bunch of starving moths.

Reply:
Based on a misunderstanding of how the new system works. Moths can eat regular foods (thanks to `components: - Food` below) _and_ MothFood. Disregard the above.